Aspen-Discovery/aspen-discovery
# Get Patron Type Object Check

Account for patron types being '0' when doing boolean check to get the patron type object (common for Sierra libraries) This issue was made apparent when attempting to enable materials requests for patrons of this type

Tested on my local

To Test:
Create a patron type 0
Enable materials requests for patrons of that type Check to see if materials requests are available via masquerade for a patron of that type
